std/xml.d

```diff
@@ -1288,14 +1288,14 @@ class Comment : Item
      * You should rarely need to call this function. It exists so that Comments
      * can be used as associative array keys.
      */
-    override size_t toHash() const nothrow { return hash(content); }
+    override size_t toHash() scope const nothrow { return hash(content); }
 
     /**
      * Returns a string representation of this comment
      */
-    override string toString() const @safe pure nothrow { return "<!--" ~ content ~ "-->"; }
+    override string toString() scope const @safe pure nothrow { return "<!--" ~ content ~ "-->"; }
 
-    override @property @safe @nogc pure nothrow bool isEmptyXML() const { return false; } /// Returns false always
+    override @property @safe @nogc pure nothrow scope bool isEmptyXML() const { return false; } /// Returns false always
```

Comment:
> This is getting out of hand

The whole std.xml needs to go anyway. If it was a templated object, most of these attributes would simply be inferred.